spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -428,13 +432,15 @@ The following table includes the available Task Application Starters based on Sp
 
 [width="100%",frame="topbot",options="header"]
 |======================
-|Artifact Type |Milestone Release |SNAPSHOT Release
+|Artifact Type |Latest Release|Milestone Release |SNAPSHOT Release
```

Comment:
It'd be good to change the column headers to be consistent for both stream and tasks.

> |Latest Release|Stable Release |SNAPSHOT Release

... probably is better